scripts/helpers/db-test-helpers.ts

```diff
@@ -65,6 +65,9 @@ export async function recreateTestDbTemplate(
     dbConfig.dbLoginPassword,
     dbConfig.dbOwner,
     dbConfig.dbOwnerPassword,
+    undefined,
```

Comment:
Since env owner main use case is in multitenant services and atm this is not used in media template - please drop `dbEnvOwner` and `dbEnvOwnerPassword` parameters from the `runResetQueries` function. If there is ever a need to introduce such roles to some media template service - it can be done at that point in time